docker/.env.example

```
@@ -197,7 +201,11 @@ EXPOSE_GRAFANA_PORT=3001
 # =============================================================================
 # Monitoring credentials
 GRAFANA_ADMIN_USER=admin
-GRAFANA_ADMIN_PASSWORD=securepassword
+# Required in production; generated by scripts/deploy.sh.
+GRAFANA_ADMIN_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the GRAFANA_ADMIN_USER key

(UnorderedKey)